app/views/oralReport/page6.scala.html

```diff
@@ -11,93 +11,52 @@
+                <li>include information on protective factors that may support desistance
+                </li>
+                <li>provide an explanation of the offender's motivation and readiness to change
+                </li>
+                <li>use bullet points to outline key information
+                </li>
+            </ul>
+
+            <p>You should also consider:</p>
+            <ul>
+                <li>diversity issues in terms of the offender’s own experiences or protected characteristics
+                </li>
+                <li>the person’s circumstances, particularly those linked to offending behaviour
+                </li>
+                <li>factors which may not be directly linked but cause difficulties for the person and impact on their desistance from crime
+                </li>
+            </ul>
         </div>
     </details>
 
     <div class="govuk-form-group@if(reportForm.allErrors.stream.anyMatch(_.key.contains("issue"))) { govuk-form-group--error}">
         <fieldset class="govuk-fieldset">
-            <legend class="govuk-fieldset__legend govuk-label">Select the relevant options</legend>
+            <legend class="govuk-fieldset__legend govuk-label">Select any factors relating to offending behaviour and the individual’s need, including any protective factors.</legend>
             <div class="govuk-checkboxes" data-module="checkboxes">
-                @checkbox(reportForm("issueAccommodation"), '_label -> "Accommodation")
-                @checkbox(reportForm("issueEmployment"), '_label -> "Employment")
-                @checkbox(reportForm("issueFinance"), '_label -> "Finances")
-                @checkbox(reportForm("issueRelationships"), '_label -> "Relationships")
-                @checkbox(reportForm("issueSubstanceMisuse"), '_label -> "Substance misuse")
-                @checkbox(reportForm("issueHealth"), '_label -> "Physical & mental health")
-                @checkbox(reportForm("issueBehaviour"), '_label -> "Thinking & behaviour")
-                @checkbox(reportForm("issueMaturity"), '_label -> "Maturity")
+                @checkboxDetailsField(reportForm, "issueAccommodation", "Accommodation", "Provide a brief assessment of accommodation")
+                @checkboxDetailsField(reportForm, "issueEmployment", "Employment, training and education", "Provide a brief assessment of employment, training and education")
+                @checkboxDetailsField(reportForm, "issueFinance", "Finances", "Provide a brief assessment of finances")
+                @checkboxDetailsField(reportForm, "issueRelationships", "Relationships", "Provide a brief assessment of relationships")
+                @checkboxDetailsField(reportForm, "issueSubstanceMisuse", "Substance misuse", "Provide a brief assessment of substance misuse")
+                @checkboxDetailsField(reportForm, "issueHealth", "Physical & mental health", "Provide a brief assessment of physical & mental health")
```

Comment:
"Experience of trauma" and "Caring Responsibilities" are on the original report template but have been omitted on the Word document.

I've checked with Alastair and they should be on here, can you include them in the same way here?

Reply:
Yeah I was just following the word doc and removed them as they weren't there.

I'll add them back in